src/test/scala/org/bitcoins/core/util/CryptoTestUtil.scala

```diff
@@ -9,7 +9,7 @@ import org.bitcoins.core.crypto.{ECPrivateKey}
  */
 trait CryptoTestUtil {
   def privateKeyBase58 = "cVLwRLTvz3BxDAWkvS3yzT9pUcTCup7kQnfT2smRjvmmm1wAP6QT"
-  def privateKeyBytes  = Base58.decode(privateKeyBase58)
+  def privateKeyBytes  = Base58.decodeCheck(privateKeyBase58).get
```

Comment:
Wasn't sure about this b/c `decodeCheck` doesn't return the checksum, but I also noticed `privateKeyBytes` and `privateKeyHex` aren't even used in our code as I didn't find any usages. All tests passed when I deleted lines 12/13 so they could be removed.